.github/workflows/main.yml

```diff
@@ -0,0 +1,23 @@
+name: Build and Deploy
+on:
+  push:
+    branches:
+      - main
+  pull_request: {}
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout 🛎️
+        uses: actions/checkout@v2.3.1
+        with:
+          submodules: 'true'
+
+      - name: Set up Yarn cache
+        uses: c-hive/gha-yarn-cache@v2
+
+      - name: Install and Build 🔨
+        run: |
+          yarn
+          yarn ci
```

Comment:
Thanks for adding CI to it!

Reply:
No Problem! Basically it looks similar to `npm run ci`! This script will validate & checks the packages, and finally test build the package to see if it is successful build or not.

Reply:
nice!